e2e-tests/functions

```diff
@@ -622,3 +623,38 @@ kubectl_bin() {
     rm "$LAST_OUT" "$LAST_ERR"
     return ${exit_status}
 }
+
+patch_secret() {
+    local secret=$1
+    local key=$2
+    local value=$3
+    
+    kubectl patch secret $secret -p="{\"data\":{\"$key\": \"$value\"}}"
+}
+
+getSecretData() {
+    local secretName=$1
+    local dataKey=$2
+    local  data=`kubectl get secrets/${secretName} --template={{.data.${dataKey}}} | base64 -D`
+    echo "$data"
+}
+
+check_mongo_auth() {
+    local uri="$1"
+
+    ping=$(run_mongo "db.runCommand({ ping: 1 }).ok" "$uri" "mongodb" "" "--quiet" | egrep -v 'I NETWORK|W NETWORK|Error saving history file|Percona Server for MongoDB|connecting to:|Unable to reach primary for set|Implicit session:|versions do not match')
+    desc "ping return"
+    if [ "${ping}" != "1" ]; then
+        return 1
+    fi
+}
+
+wait_cluster_consistency() {
+    cluster_name=$1
+
+    sleep 7 # wait for two reconcile loops ;)  3 sec x 2 times + 1 sec = 7 seconds
+    until [[ "$(kubectl_bin get psmdb "${cluster_name}" -o jsonpath='{.status.state}')" == "ready" ]]; do
+        echo 'waiting for cluster readyness'
+        sleep 20
+    done
```

Comment:
looks like tests could be stuck forever here

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/387/commits/07c77f8b2b1443890f3a279aae242bb9a6ad1c44